# Faryabi/Vahedi Laboratories at the University of Pennsylvania

# When and where:

We meet on **Fridays** at **4pm** in **301 BRB** (421 Curie Blvd). Light snack is provided thanks to the generosity of the Institute for Immunology (IFI). 

----

**Journal club schedule**

----
**2018**

----

May 18th: **3D genome organization review session**. Ben Auerbach. Vahedi lab. The role of transcription factor AP-1 on multi-loop interaction communities:
https://www.sciencedirect.com/science/article/pii/S1097276517306032?via%3Dihub

----

May 11th: **Single-cell RNA-seq review session**. Sasi Manne. Wherry lab. 
SIMLR-https://www.nature.com/articles/nmeth.4207
https://github.com/BatzoglouLabSU/SIMLR

SCENIC-https://www.nature.com/articles/nmeth.4463
https://github.com/aertslab/SCENIC

----

May 4th: **3D genome organization review session**. No JC.

----

April 27th: **Single-cell RNA-seq review session**. Gregory Schwartz. Faryabi lab. Modularity maximization in community detection:
https://journals.aps.org/pre/abstract/10.1103/PhysRevE.84.066122

----

April 20th: **3D genome organization review session**. No JC.

----

April 13th: **Single-cell RNA-seq review session**. Cancelled.

----

April 6th: **3D genome organization review session**. John Johnson. Vahedi lab: 3D genome during T cell development: https://www.ncbi.nlm.nih.gov/pubmed/29466755

----

March 30th: **Single-cell RNA-seq review session**. Zhicheng Ji and Weiqiang Zhou. Trainees from Johns Hopkins: TSCAN, SCRAT and BIRD.
Slides:

https://drive.google.com/file/d/1lJP4SMeM-H7VBPZ9R3Bq6zFbBsofOJKx/view?usp=sharing

https://drive.google.com/file/d/1B12gdqleDz5xaJEXls_Ei3GoRBI9VO1V/view?usp=sharing

----

March 23th: Floor Happy Hour. No JC.

----

March 16th: **Single-cell RNA-seq review session**. Stanley Cai, Vahedi lab, BISCUIT: https://genomicscomputbiol.org/ojs/index.php/GCB/article/view/46 

----

March 9th: **3D genome organization review session**. Faryabi lab. https://www.ncbi.nlm.nih.gov/pubmed/29224777

----

March 2nd: **Single-cell RNA-seq review session**. Jonathan Schug (NGSC Director) and Kaestner lab:
https://www.nature.com/articles/nbt.3854

----
**2017**

----

November 17th 2017: Naomi Goldman; Kees Murre's new Cell paper: https://www.ncbi.nlm.nih.gov/pubmed/28938112

----

November 10rd, 2017: Stanley Cai; PhenoGraph for single-cell RNA-seq developed by Dana Pe'er:
http://www.cell.com/cell/abstract/S0092-8674(15)00637-6

----

November 3rd 2017: Maria Fasolino; two independent modes of chromatin organization: https://www.nature.com/nature/journal/vaap/ncurrent/pdf/nature24281.pdf

----

October 20st, 2017: 3rd Floor Happy Hour!

----

October 13th, 2017: Penn Epigenetics Institute Retreat.

----

October 6th, 2017: IGG Retreat.

----

September 29th, 2017: Stanley Cai, Monocle 2, "Reversed graph embedding resolves complex single-cell trajectories": http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4402.html

----

September 22nd, 2017: Yeqiao Zhou, "Enhancer connectome in primary human cells reveals target genes of disease-associated DNA elements", HiChIP in T cells, http://www.biorxiv.org/content/early/2017/08/30/178269 

----

May 5th, 2017: Yeqiao Zhou, HiChIP party!

HiC-Pro -- raw data processing: 
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0831-x

ICE -- contact matrix normalization:
http://www.nature.com/nmeth/journal/v9/n10/full/nmeth.2148.html

Fit-Hi-C -- call significant contact: 
http://genome.cshlp.org/content/early/2014/02/05/gr.160374.113.full.pdf+html


----

April 27th, 2017: No JC;

----

April 21th, 2017: No JC;

----

April 14th, 2017: Matt Paul; single-cell RNA-seq analysis using Census: http://www.nature.com/nmeth/journal/v14/n3/full/nmeth.4150.html

----

April 7th, 2017: No JC;

----

March 31th, 2017: Sasi Manne; http://www.pnas.org/content/early/2017/01/25/1621150114.abstract

----

March 24th, 2017: Austin Huffman; scRNA-seq in CD8 T cells: http://www.nature.com/ni/journal/vaop/ncurrent/full/ni.3688.html

----

March 17th, 2017: Megan Frederick; Dumpter diving in RNA-seq: http://biorxiv.org/content/early/2016/05/13/053041

----

March 10th, 2017: GW; single-cell ATAC analysis
http://biorxiv.org/content/biorxiv/early/2017/02/21/110346.full.pdf

----

March 3rd, 2017: No JC.

----

Feb. 24th, 2017: IGG recruitment. 

----

Feb. 17th, 2017: Yeqiao Zhou; http://www.nature.com/ng/journal/v48/n3/full/ng.3487.html

----

Feb. 10th, 2017: GCB recruitment. 

----

Feb. 3rd, 2017: GGR recruitment. 

----

Jan. 27th, 2017: Golnaz Vahedi; Quantitative modeling of transcription factor binding
specificities using DNA shape http://rohslab.cmb.usc.edu/Papers/2015_Pnas_Zhou.pdf



----

Jan. 20th, 2017: GGR recruitment. 

----

Jan. 13th, 2017: Greg Grant; Comprehensive benchmarking of RNA-seq aligners-http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.4106.html

----
**2016**
----

Dec 9th: Steph Sansbury-https://f1000research.com/articles/4-1521/v2

----

Nov 18th: no JC.

----

Nov 11th: Sasi Manne-http://www.nature.com/nbt/journal/v32/n4/full/nbt.2859.html

----

Nov 4th:Derek Kelly-http://science.sciencemag.org/content/sci/352/6285/600.full.pdf

----

October 28th:Yeqiao. http://www.sciencedirect.com/science/article/pii/S1074761315005324

----

October 21th:IGG Retreat. no JC.

----

October 14th:Golnaz Vahedi-http://science.sciencemag.org/content/352/6282/189.full.pdf+html

----

October 7th:GW-http://www.biomedcentral.com/qc/1752-0509/9/S1/S7/

-----


September 23rd: Babak Faryabi-Proteogenomics: http://www.cell.com/fulltext/S0092-8674(16)30913-8

----

September 16th:George Gerogakilas: miRNA prediction tools.

-----

September 9th-Cameron McConkey: Single-cell TCR sequencing-http://www.nature.com/nmeth/journal/v13/n4/abs/nmeth.3800.html

Protocol: https://www.dropbox.com/s/l3uvdepoldnzb9b/C1%20Tracer%20Workflow.pdf?dl=0

Slides: https://www.dropbox.com/s/3wyardm3m0m6rtz/Tracer%20Presentation.pdf?dl=0

-----

Summer break-July 15-August 26th. We will resume our JC in the fall.

-----

July 15th-JJ: Single-cell ATAC-seq: http://www.nature.com/nature/journal/v523/n7561/full/nature14590.html

-----

July 1st-Yeqiao Zhou

http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-9-559

-----

June 24th-No JC

-----

June 17th-GW: Proteogenomics: http://www.nature.com/nature/journal/v534/n7605/full/nature18003.html

-----

June 10th-Sasi Manne

-----

June 3th-Kasia Katarzyna and Simone Sidoli on Proteomics-

Slides: https://www.dropbox.com/s/ut9yqan948ft2tf/Journal%20club%20-%20April%202016.pptx?dl=0

-----

May 27th-Kasia Katarzyna and Simone Sidoli on Proteomics-

-----

May 13th-Jeff's prelim practice-

-----

May 6th-no JC

-----

April 29th: Kasia Katarzyna and Simone Sidoli on Proteomics-

Slides on an intorduciton to proteomics: https://www.dropbox.com/s/ut9yqan948ft2tf/Journal%20club%20-%20April%202016.pptx?dl=0

-----

April 22nd: Golnaz Vahedi- http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3539.html

Slides on ensemble learning and TargetFinder: https://www.dropbox.com/s/sj7jv9klw8mpnb2/April22_2016.pptx?dl=0

-----

April 15th: no JC.

-----

April 8th: no JC. 

-----

April 1st: Babak Faryabi on Cancer Genomics.

-----

March 25th: Jeff Lin-http://genome.cshlp.org/content/21/3/447

Slides on CENTIPEDE: https://www.dropbox.com/s/u1x2zi7jcycnsuo/032516%20CENTIPEDE.pptx?dl=0

-----

March 18th: Sasi Manne-http://nar.oxfordjournals.org/content/early/2013/09/25/nar.gkt850.full

Slides on footprinting using Wellington: https://www.dropbox.com/s/3xezaq2yv252vr8/JC_SM_03182016.ppt?dl=0

-----

March 11th: John Johnson-http://m.genome.cshlp.org/content/early/2015/08/27/gr.192294.115

Slides on NucleoATAC: https://www.dropbox.com/s/87uvrtvbq93ilyd/Computational_JC_NucleoATAC.pptx?dl=0

-----

March 4th: Gregory Schwartz-http://ieeexplore.ieee.org/xpl/login.jsp?reload=true&tp=&arnumber=5767835&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D5767835

Slides on spectral clustering: https://www.dropbox.com/s/0hwzu4g9e21gh8f/GW_clustering.pdf?dl=0

-----

Feb. 19th: Hackathon meeting-https://github.com/VahediLab/IFIGeneticsHackathon_2016

-----

Feb. 5th: Yeqiao Zhou-http://www.genomebiology.com/2014/15/12/550

Slides on DESeq2: https://www.dropbox.com/s/i2bbpsjlbiqgr4n/160205_DESeq2.pdf?dl=0

-----

January 29th: Sepideh Dolatshahi- http://www1.cs.columbia.edu/~dpeer/pub/stke2005.pdf

Slides on Bayesian networks: https://www.dropbox.com/s/zueacj7v2he66xz/BayesianNets_Sepideh.pptx?dl=0

-----

January 22nd: Golnaz Vahedi-http://www.nature.com/nprot/journal/v9/n6/full/nprot.2014.083.html

Slides on motif analysis focusing on MEME suite: https://www.dropbox.com/s/xmcoe5sw8hew8j5/Motif_overview.pptx?dl=0

-----

January 15th (at 5pm, BRB 305): Babak Faryabi- http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3471.html

Slides on clustering and posterior probability:: https://www.dropbox.com/s/zwmrj3mehvzhhx4/CompJC_160111.pptx?dl=0

-----
#Reading recommendations:

http://statweb.stanford.edu/~tibs/ElemStatLearn/

http://www-bcf.usc.edu/~gareth/ISL/

